Occasional failure of StreamsSpec

```
[info] - should call its response handler when there is a response whether failed or not *** FAILED ***
[info]   java.lang.AssertionError: assertion failed: expected HttpResponse(404 Not Found,List(),HttpEntity.Strict(none/none,ByteString()),HttpProtocol(HTTP/1.1)), found Success(Done)
```

--- a/akka-sse/src/test/scala/de/heikoseeberger/akkasse/pattern/StreamsSpec.scala
+++ b/akka-sse/src/test/scala/de/heikoseeberger/akkasse/pattern/StreamsSpec.scala
@@ -77,8 +77,12 @@ class StreamsSpec extends BaseSpec with ScalaFutures with EventStreamMarshalling
       testSource.sendNext(httpResponse)
       testSource.sendComplete()
 
-      testObserver.expectMsg(httpResponse)
-      testObserver.expectMsg(Success(Done))
+      testObserver.expectMsgPF() {
+        case `httpResponse` =>
+          testObserver.expectMsg(Success(Done))
+        case Success(Done) =>
+          testObserver.expectMsg(httpResponse)
+      }
     }
 
     "not call its response handler when failed" in {
@@ -102,8 +106,12 @@ class StreamsSpec extends BaseSpec with ScalaFutures with EventStreamMarshalling
       testSource.sendNext(httpResponse)
       testSource.sendComplete()
 
-      testObserver.expectMsg(httpResponse)
-      testObserver.expectMsg(Success(Done))
+      testObserver.expectMsgPF() {
+        case `httpResponse` =>
+          testObserver.expectMsg(Success(Done))
+        case Success(Done) =>
+          testObserver.expectMsg(httpResponse)
+      }
     }
 
     "pass through all non heartbeat events" in {

